### PR TITLE
Feature/new desktop view

### PR DESCRIPTION
**Pull request by Kovler10**
missing features:
- closing cards with X button
- creating communities

### CODE REVIEW

File: apps/ui/src/Pages/RidesFeed/RidesFeed.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  rides,
+  communities,
+  userRidesStatus,
+  userCommunities,
+  setSelectedRide,
+}: RidesFeedProps) => {
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+
+  return (
+    <Page sx={{ width: '100%' }}>
```
Comment (Zoeshwa) [inline apps/ui/src/Pages/RidesFeed/RidesFeed.tsx:32]: same

File: apps/ui/src/Components/Rides/RideList/RideList.tsx [hunk 2]
```diff
@@ -1,43 +1,53 @@
...
+    if (setOpenDialog) {
+      setOpenDialog(isCreateRideDialog);
+    }
+  }, [isCreateRideDialog]);
 
-  const handleAddClick = (_event: React.MouseEvent) => {
-    setIsCreateRideDialogOpen(true);
-  };
+  useEffect(() => {
+    if (!!openDialog) {
```
Comment (Zoeshwa) [inline apps/ui/src/Components/Rides/RideList/RideList.tsx:44]: is this !! necessary?

File: apps/ui/src/Components/Rides/RideList/RideList.tsx [hunk 3]
```diff
@@ -1,43 +1,53 @@
...
+      setIsCreateRideDialogOpen(openDialog);
+    }
+  }, [openDialog]);
```
Comment (Zoeshwa) [inline apps/ui/src/Components/Rides/RideList/RideList.tsx:47]: Is this the best way? Looks a little weird the 2 useEffects

File: apps/ui/src/Communities/CommunitiesFeed.tsx [hunk 4]
```diff
@@ -203,17 +208,13 @@ const CommunitiesFeed = () => {
...
             userStatusIsLoading={userStatusIsLoading}
-            userId={user?.id ?? DEFAULT_USER_ID}
+            userStatus={userCommunitiesStatus[community.id]}
           />
         ))}
       </FeedList>
-      <AddNewButton
-        handleAddClick={handleAddClick}
-        tooltipText="Create a new community"
-      />
```
Comment (Zoeshwa) [inline apps/ui/src/Communities/CommunitiesFeed.tsx:216]: where is the add button

File: apps/ui/src/DesktopApp/home/home.tsx [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+              })),
+            ),
+          ) as MarkerInfo[]
+        }
+      />
+    </Page>
+  );
+};
+
+export { Home };
```
Comment (Zoeshwa) [inline apps/ui/src/DesktopApp/home/home.tsx:213]: Feels that there are double things here that can be combined with the display of the mobile

**Review by Zoeshwa — CHANGES_REQUESTED** [review]
Cool, I left a few comments, maybe I didn't fully understand the implementation, so if they are unnecessary you can ignore them

File: apps/ui/src/Communities/CommunitiesFeed.tsx (hunk 4 above)
Comment (Kovler10) [reply apps/ui/src/Communities/CommunitiesFeed.tsx:216]: next to the My communities filter

**Review by Zoeshwa — APPROVED** [review]

**Pull request merged**